docs/create-component.md

```diff
@@ -0,0 +1,72 @@
+# Create Component Script
+
+The create-component script is a utility for generating new React components(React Native coming soon!) in the MetaMask Design System with consistent structure and boilerplate code.
```

Comment:
We can update this documentation to include React Native once the script has been created for it. Have added (React Native coming soon!) in this sentence